cms/standard_pages/models.py

```diff
@@ -121,6 +124,11 @@ def _get_formatted_child_pages(self, request: "HttpRequest") -> list[dict[str, d
                         "url": child_page.get_url(request=request),
                     },
                     "description": getattr(child_page, "listing_summary", "") or getattr(child_page, "summary", ""),
+                    "metadata": get_document_metadata(
+                        get_content_type_for_page(child_page),
+                        child_page.specific_deferred.publication_date,
+                        _("Released"),
```

Comment:
with this being the same, let's make it the default on `get_document_metadata`, and avoid repetition here and the base block.

other than that, LGTM